components/PersonView/PersonView.jsx

```diff
@@ -0,0 +1,109 @@
+import { useState, useEffect } from 'react';
+import { getResident } from 'utils/api/residents';
+
+const PersonView = ({ personId }) => {
+  const [person, setPerson] = useState();
+
+  const getPerson = async (personId) => {
+    try {
+      const data = await getResident(personId);
+      setPerson(data);
+    } catch {
+      console.log('error');
+    }
+  };
+  useEffect(() => {
+    getPerson(personId);
+  }, [personId]);
+
+  return (
+    <>
+      <h1>View person</h1>
+      <h2 className="personDetails">PERSON DETAILS</h2>
+      <hr className="personDetailsHr" />
+      <dl className="govuk-summary-list--no-border   lbh-summary-list">
+        <div className="govuk-grid-row">
+          <div className="govuk-grid-column-one-half">
+            <dt
+              className="govuk-summary-list__key"
+              style={{ display: 'inline-flex' }}
+            >
+              {person?.firstName} {person?.lastName}
+            </dt>
+            <dd className="govuk-summary-list__value"></dd>
+          </div>
+          <div className="govuk-grid-column-one-half">
+            <dt className="govuk-summary-list__key">Mosaic ID:</dt>
+            <dd className="govuk-summary-list__value">#{person?.mosaicId}</dd>
+          </div>
+        </div>
+        <hr />
+
+        <div className="govuk-grid-row">
+          <div className="govuk-grid-column-one-half">
+            <dt className="govuk-summary-list__key">Date of birth</dt>
+            <dd className="govuk-summary-list__value">
+              {new Date(person?.dateOfBirth).toLocaleDateString('en-GB')}
+            </dd>
+          </div>
+          <div className="govuk-grid-column-one-half">
+            <dt className="govuk-summary-list__key">NHS number</dt>
+            <dd className="govuk-summary-list__value">{person?.nhsNumber}</dd>
+          </div>
+        </div>
+
+        <div className="govuk-grid-row">
+          <div className="govuk-grid-column-one-half">
+            <dt className="govuk-summary-list__key">Nationality</dt>
+            <dd className="govuk-summary-list__value">{person?.nationality}</dd>
+          </div>
+        </div>
+
+        <div className="govuk-grid-row">
+          <div className="govuk-grid-column-one-half">
+            <dt className="govuk-summary-list__key">Gender</dt>
+            <dd className="govuk-summary-list__value">{person?.gender}</dd>
+          </div>
+        </div>
+
+        <div className="govuk-grid-row">
+          <div className="govuk-grid-column-one-half">
+            <dt className="govuk-summary-list__key">
+              <h4>Address</h4>
```

Comment:
Probably this shouldn't be `h4`

Reply:
I made it h4 as anything bigger made things not align properly. Should it be bigger or smaller?

Reply:
The problem is the design, that is not compatible (out of the box) with our component library :(